tasks/main.yml

```diff
@@ -4,6 +4,12 @@
   with_items:
   - git
 
+- name: Install socat
+  package:
+    name: socat
+    state: present
+  when: enable_onion_support is defined
```

Comment:
Ok so it work on the other part because the vars are not in defaults/ (or because there is a mismatch on the variable name). Guess that's why I should push at 4 in the morning